lib/asciidoctor-pdf/converter.rb

```diff
@@ -419,6 +419,21 @@ def convert_admonition node
     theme_margin :block, :top
     icons = node.document.attr? 'icons', 'font'
     label = icons ? (node.attr 'name').to_sym : node.caption.upcase
+    
+    valid_own_icon = false
+    if !icons        
+        iconsdir = node.attr 'iconsdir'
+        iconname = node.attr 'icon'
+        if iconname.nil? or iconname.blank?
+            iconname = (node.attr 'name')+ ".png"
```

Comment:
There's already built-in support for the icon file extension, so we should support that. Here's the relevant code from the built-in HTML converter:

https://github.com/asciidoctor/asciidoctor/blob/v1.5.4/lib/asciidoctor/abstract_node.rb#L267-L278